src/main/webapp/map.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Map Project</title>
+
+  <!-- API key access -->
+  <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyDDipc0Fcrm1Zt2d1Ig7xDylFNgpxiqF3s"></script>
+
+  <!-- renders map and adds marker -->
+  <script>
```

Comment:
I suggest moving the script sections to their own files. See the other html files in the "webapp" folder for what that looks like. It'a up to you though.

Reply:
Just moved the scripts to a separate map-loader.js file!